### DISCUSSION TITLE
Mining intensity parameter

### DISCUSSION THREAD

**Discussion by cypher619** [Lolliedieb/lolMiner-releases#1303] [Ideas]
Is it possible to add a mining intensity parameter so that people who mine on their daily driver computers wont be affected by the mining process.  For me every now and then my display fails to keep up and sometimes pauses and then rapidly catches up.

**Comment by babacakka** [comment]
When you're mining on a regular PC with only one GPU you can surf the web at the same time but never try to watch movies or try to work with large photos or any VGA memory intense applications... 

Because the VGA card at that time is at the "Compute Mode" and gives all its power to mining process...
